### PR TITLE
Include wsl vm addr

### PR DESCRIPTION
**Pull request by Nino-K**

### CODE REVIEW

File: cmd/rancher-desktop-guestagent/main.go [hunk 1]
```diff
@@ -154,3 +161,27 @@ func tryConnectDocker(ctx context.Context, verify func(context.Context) error) e
 		}
 	}
 }
+
+// Gets the wsl interface address by doing a lookup by name for 'eth0'.
```
Comment (mook-as) [inline cmd/rancher-desktop-guestagent/main.go]: Technically, via the given interface. It's just that the input happens to be constant.

File: cmd/rancher-desktop-guestagent/main.go [hunk 2]
```diff
@@ -78,8 +81,12 @@ func main() {
 		}
 
 		group.Go(func() error {
-			forwarder := forwarder.NewVtunnelForwarder(*vtunnelAddr)
-			portTracker := tracker.NewPortTracker(forwarder)
+			wslAddr, err := getWSLAddr(wslInfName)
+			if err != nil {
+				return err
+			}
+			forwarder := forwarder.NewVtunnelForwarder((*vtunnelAddr))
```
Comment (mook-as) [inline cmd/rancher-desktop-guestagent/main.go]:
```suggestion
			forwarder := forwarder.NewVtunnelForwarder(*vtunnelAddr)
```
It came back…

**Review by mook-as — APPROVED** [review]
Just a couple trivial changes.

**Comment by Nino-K** [comment]
https://github.com/rancher-sandbox/rancher-desktop/issues/2677

**Pull request merged**